cmd/updater/main.go

```diff
@@ -183,7 +184,17 @@ func main() {
 		fixers = append(fixers, updater.FixPersistent(log, client, path, ticker.C))
 	}
 
-	if err := updater.Update(ctx, client, mets, opt.config, opt.gridPrefix, opt.groupConcurrency, opt.groups.Strings(), groupUpdater, opt.confirm, opt.wait, fixers...); err != nil {
+	u := updater.UpdateOptions{
+	  ConfigPath:             opt.config,
+	  GridPrefix:             opt.gridPrefix,
+	  GroupConcurrency:       opt.groupConcurrency,
+	  GroupNames:             opt.groups.Strings(),
+   Write:                  opt.confirm,
+	  Freq:                   opt.wait,
+
+ }   
+
+	if err := updater.Update(ctx, u , client, mets, groupUpdater, fixers...); err != nil {
```

Comment:
nit: options should go next-to-last (would be last, but `fixers...` has to be last)